sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -3144,5 +3144,74 @@ object Radar {
             logBuffer.write(level, message, type)
         }
     }
+}
+
+/**
+ * Autocompletes partial addresses and place names, sorted by relevance.
+ *
+ * @see [](https://radar.com/documentation/api#autocomplete)
+ *
+ * @param[query] The partial address or place name to autocomplete.
+ * @param[near] A location for the search.
+ * @param[layers] Optional layer filters.
+ * @param[limit] The max number of addresses to return. A number between 1 and 100.
+ * @param[country] An optional country filter. A string, the unique 2-letter country code.
+ * @param[callback] A callback.
+ */
```

Comment:
Add a note that this version of the function is deprecated, and to prefer the above.